src/item-view.js

```diff
@@ -105,5 +122,122 @@ Marionette.ItemView = Marionette.AbstractView.extend({
     this.$el.html(html);
 
     return this;
+  },
+
+  // Add a single region, by name, to the layoutView
+  addRegion: function(name, definition) {
+    var regions = {};
+    regions[name] = definition;
+    return this._buildRegions(regions)[name];
+  },
+
+  // Add multiple regions as a {name: definition, name2: def2} object literal
+  addRegions: function(regions) {
+    this.regions = _.extend({}, this.regions, regions);
+    return this._buildRegions(regions);
+  },
+
+  // Remove a single region from the LayoutView, by name
+  removeRegion: function(name) {
+    delete this.regions[name];
+    return this.regionManager.removeRegion(name);
+  },
+
+  // Provides alternative access to regions
+  // Accepts the region name
+  // getRegion('main')
+  getRegion: function(region) {
+    return this.regionManager.get(region);
+  },
+
+  // Get all regions
+  getRegions: function(){
+    return this.regionManager.getRegions();
+  },
+
+  // Enable easy overriding of the default `RegionManager`
+  // for customized region interactions and business specific
+  // view logic for better control over single regions.
+  getRegionManager: function() {
+    return new Marionette.RegionManager();
+  },
+
+  // Handle destroying regions, and then destroy the view itself.
+  destroy: function() {
+    if (this.isDestroyed) { return this; }
+
+    this.regionManager.destroy();
+    return Marionette.AbstractView.prototype.destroy.apply(this, arguments);
+  },
+
+  // Internal method to initialize the regions that have been defined in a
+  // `regions` attribute on this layoutView.
+  _initializeRegions: function(options) {
+    var regions;
+    this._initRegionManager();
+
+    regions = Marionette._getValue(this.regions, this, [options]) || {};
+
+    // Enable users to define `regions` as instance options.
+    var regionOptions = this.getOption.call(options, 'regions');
+
+    // enable region options to be a function
+    regionOptions = Marionette._getValue(regionOptions, this, [options]);
```

Comment:
Let's talk about refactors in a separate issue. This is a direct port of the LayoutView into the ItemView; no code changes.